<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="15,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="lineWidth">
         <number>40</number>
        </property>
        <property name="text">
         <string>演示区域</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>980</width>
           <height>68</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionSaveXml"/>
   <addaction name="actionLoadXml"/>
   <addaction name="actionResetConfig"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenPic">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveXml"/>
    <addaction name="actionLoadXml"/>
    <addaction name="actionResetConfig"/>
   </widget>
   <addaction name="menuOpenPic"/>
  </widget>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/stylesheet/info.png</normaloff>:/new/icon/stylesheet/info.png</iconset>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="actionSaveXml">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/stylesheet/error.png</normaloff>:/new/icon/stylesheet/error.png</iconset>
   </property>
   <property name="text">
    <string>保存XML</string>
   </property>
  </action>
  <action name="actionLoadXml">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/stylesheet/radio_selected.png</normaloff>:/new/icon/stylesheet/radio_selected.png</iconset>
   </property>
   <property name="text">
    <string>加载XML</string>
   </property>
  </action>
  <action name="actionResetConfig">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/stylesheet/radio_normal.png</normaloff>:/new/icon/stylesheet/radio_normal.png</iconset>
   </property>
   <property name="text">
    <string>重置配置</string>
   </property>
  </action>
  <action name="actionRaTry">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
